File: Admin_p/Admin_p.Web/Modules/ServerTypes/Identity/AspNetUsersColumns.ts
import { ColumnsBase, fieldsProxy } from '@serenity-is/corelib';
import { Column } from '@serenity-is/sleekgrid';
import { AspNetUsersRow } from './AspNetUsersRow';

export interface AspNetUsersColumns {
    Id: Column<AspNetUsersRow>;
    FirstName: Column<AspNetUsersRow>;
    CreatedAt: Column<AspNetUsersRow>;
    Role: Column<AspNetUsersRow>;
    ShippingAddress: Column<AspNetUsersRow>;
    PhoneNumber: Column<AspNetUsersRow>;
    UpdatedAt: Column<AspNetUsersRow>;
    UserName: Column<AspNetUsersRow>;
    NormalizedUserName: Column<AspNetUsersRow>;
    Email: Column<AspNetUsersRow>;
    NormalizedEmail: Column<AspNetUsersRow>;
    EmailConfirmed: Column<AspNetUsersRow>;
    PasswordHash: Column<AspNetUsersRow>;
    SecurityStamp: Column<AspNetUsersRow>;
    ConcurrencyStamp: Column<AspNetUsersRow>;
    PhoneNumberConfirmed: Column<AspNetUsersRow>;
    TwoFactorEnabled: Column<AspNetUsersRow>;
    LockoutEnd: Column<AspNetUsersRow>;
    LockoutEnabled: Column<AspNetUsersRow>;
    AccessFailedCount: Column<AspNetUsersRow>;
}

export class AspNetUsersColumns extends ColumnsBase<AspNetUsersRow> {
    static readonly columnsKey = 'Identity.AspNetUsers';
    static readonly Fields = fieldsProxy<AspNetUsersColumns>();
}

File: Admin_p/Admin_p.Web/Modules/Identity/AspNetUsers/AspNetUsersColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace Admin_p.Identity.Columns;

[ColumnsScript("Identity.AspNetUsers")]
[BasedOnRow(typeof(AspNetUsersRow), CheckNames = true)]
public class AspNetUsersColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight, EditLink]
    public string Id { get; set; }
    public string FirstName { get; set; }
    public DateTime CreatedAt { get; set; }
    public string Role { get; set; }
    public string ShippingAddress { get; set; }
    public string PhoneNumber { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string UserName { get; set; }
    public string NormalizedUserName { get; set; }
    public string Email { get; set; }
    public string NormalizedEmail { get; set; }
    public bool EmailConfirmed { get; set; }
    public string PasswordHash { get; set; }
    public string SecurityStamp { get; set; }
    public string ConcurrencyStamp { get; set; }
    public bool PhoneNumberConfirmed { get; set; }
    public bool TwoFactorEnabled { get; set; }
    public DateTimeOffset LockoutEnd { get; set; }
    public bool LockoutEnabled { get; set; }
    public int AccessFailedCount { get; set; }
}